seqr/views/apis/dataset_api_tests.py

```diff
@@ -63,6 +67,7 @@ def test_add_variants_dataset(self, mock_random):
 
         mock_random.return_value = 98765432101234567890
 
+        mock_analyst_access.return_value = False
```

Comment:
there are actual test fixture projects that do and do not have this access. Use the correct fixture data, do not mock what are essentially database values